back/app/models/custom_field.rb

```diff
@@ -182,6 +182,20 @@ def accept(visitor)
   def title_multiloc
     if code == 'ideation_section1'
       input_term = resource.participation_context.input_term || ParticipationContext::DEFAULT_INPUT_TERM
+
+      phases = resource.participation_context.phases
+      if phases.present?
+        now = Time.zone.now
+        ideation_types = %w[ideation budgeting]
+        ideation_phases = phases.select { |phase| ideation_types.include? phase.participation_method }
+        current_phase = ideation_phases.each.detect { |phase| phase.start_at <= now && now <= phase.end_at }
```

Comment:
We have helper methods in the `ParticipationContextService` and `TimelineService` to get the participation context.

Reply:
OK, cool someone had already done the hard work! I'll make those changes

Reply:
OK, so the timeline service didn't quite have the methods I needed so I've actually added a new method to the timeline service. Better place to put the logic though, and makes it clearer.